src/m365/purview/commands/retentionevent/retentionevent-add.spec.ts

```diff
@@ -88,9 +88,9 @@ describe(commands.RETENTIONEVENT_ADD, () => {
   let commandInfo: CommandInfo;
 
   before(() => {
-    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
-    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
-    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    sinon.stub(auth, 'restoreAuth').resolves();
+    sinon.stub(telemetry, 'trackEvent').returns();
+    sinon.stub(pid, 'getProcessName').returns('');
     auth.service.connected = true;
```

Comment:
Seems like we're missing `sinon.stub(session, 'getId')` here. Could you add it please?